src/odemis/acq/test/fastem_conf_test.py

```diff
@@ -74,8 +75,13 @@ def test_configure_scanner_overview(self):
         self.assertGreater(self.scanner.horizontalFoV.value, 1.e-3)  # should be big FoV for overview
         self.assertEqual(self.scanner.rotation.value, math.radians(5))
 
+        scanner_md = self.scanner.getMetadata()
+
+        # check that the MD_POS_COR is correctly set for overview imaging.
```

Comment:
Another test you could add is that the fallback is correctly working in case there is no MD provided. But maybe not absolutely necessary.

Reply:
I'm not able to remove the MD_FIELD_FREE_POS_SHIFT from the metadata, so I don't think there is a way to test this :(.